android/app/google-services-old.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file has sensitive information in it. It should not be committed to a public git repo 😬

Reply:
I just realised one is "old". I guess that means there was one there already.. can we remove the new one from git?